src/QMCWaveFunctions/tests/test_rotatedspos.cpp

```diff
@@ -0,0 +1,305 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Also please keep upper case file name.
test_RotatedSPOs.cpp to match RotatedSPOs class.

Reply:
Fixed.